mountpoint-s3/src/fuse/session.rs

```diff
@@ -89,8 +92,29 @@ impl FuseSession {
     /// Block until the file system is unmounted or this process is interrupted via SIGTERM/SIGINT.
     /// When that happens, unmount the file system (if it hasn't been already unmounted).
     pub fn join(mut self) -> anyhow::Result<()> {
-        let msg = self.receiver.recv();
-        trace!("received message {msg:?}, closing filesystem session");
+        let mut sys =
+            System::new_with_specifics(RefreshKind::new().with_processes(ProcessRefreshKind::new().with_memory()));
```

Comment:
Is this retrieving memory info for all processes? In any case, it seems to be redundant if you then use `refresh_process_specifics` on line 111.

Reply:
Yes, this will pre-fill memory info for all processes then we can refresh a specific one later. The performance might be a bit different, but creating an empty `System` should be fine too.